x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/ml/trained_model_cat_apis.yml

```
@@ -78,17 +79,17 @@ setup:
         model_id: a-regression-model-0
   - match:
       $body: |
-        / #id                           heap_size    operations    create_time    ingest.pipelines  data_frame.id
-        ^ (a\-regression\-model\-0 \s+  \w+      \s+ \d+       \s+ .*?        \s+ \d+               .*?          \n)+  $/
+        / #id                           heap_size    operations    create_time  type  ingest.pipelines  data_frame.id
+        ^ (a\-regression\-model\-0 \s+  \w+      \s+ \d+       \s+ .*?        \s+ \d+               .*?    \n)+  $/
```

This doesn't seem to actually check if type is returned.

I think you need to add another regex check
```suggestion
        ^ (a\-regression\-model\-0 \s+  \w+      \s+ \d+       \s+ .*?        \s+  \w+  \d+               .*?    \n)+  $/
```